enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -319,6 +318,19 @@
         provided    (update-keys params #(api/check-400 (param-id (name %)) "Unexpected parameter provided"))]
     (actions/execute-action! action (merge row-params provided))))
 
+;; These don't belong under /data-editing, but for now we have them nested here to avoid adding a new premium feature.
+
+(api.macros/defendpoint :post "/actions/v2/:action-id/describe" :- [:map
```

Comment:
maybe it should be normal actions api and won't need premium feature?

Reply:
These are going to be more powerful so I think it's reasonable to pay for them? Let's revisit when we stabilize?